pom.xml

```diff
@@ -102,22 +129,36 @@
             <version>${hamcrest-all-version}</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
+       <dependency>
             <groupId>com.github.stefanbirkner</groupId>
             <artifactId>system-rules</artifactId>
             <version>${system-rules-version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>uk.org.webcompere</groupId>
+            <artifactId>system-stubs-core</artifactId>
+            <version>2.1.6</version>
```

Comment:
could this version be moved to the properties above

Reply:
Done